### DISCUSSION TITLE
Global settings for projects

### DISCUSSION THREAD

**Discussion by Andre601** [modrinth/.github#16] [Feature requests]
### Is your feature request related to a problem? Please describe.

Whenever you create a new project, you must fill in the same standard information such as Discord Invite or donation links.
This is extremely time-consuming and can even be frustrating.

### Describe the solution you'd like

There should be an option to save specific information that could then be used whenever a new project is created by the user.

Perhaps a dummy project that would allow you to set all the info to then use in new projects.

### Describe alternatives you've considered

Writing the same info into the same fields over and over and over again...

**Comment by jadelily18** [comment]
this would be awesome but probably requires backend support too (i think??)

**Comment by jadelily18** [comment]
maybe you could create/import/export templates in a json format and just set a default in your user settings? i feel like this would be a really useful feature for a lot of people

**Comment by Andre601** [comment]
Perhaps an interesting solution could be to use the existing project creation thing where you can create a template to load for other projects later on. Maybe with a specific name or smth.
Like similar to how f.e. GitHub treats specific repositories different based on the names they have.

In that case could you f.e. have a project named like your username act as the template.

Alternatively could we have some links in the user profile itself and then pull those on project creation to auto-fill. I think people would like to have their GitHub, Patreon, etc. linked on their profile in Modrinth.

**Comment by triphora** [comment]
I've linked #827 to this because it adds the ability to bulk-edit links.

**Comment by Minenash** [comment]
While the bulk editing thing is indeed awesome, it doesn't solve this issue unless your publishing multiple mods at the same time, you still need to type the linked manually for each mod you make. If you're publishing 10 at once, it does help a lot, but if time publishing 1 every month are so, not really

**Comment by triphora** [comment]
Could a separate issue be made for templates?

**Comment by Andre601** [comment]
> Could a separate issue be made for templates?

To be honest: No.

You were a bit to eager closing this, as the main feature request is imo still open here.
I never suggested this feature of bulk editing but a feature to have options already filled out when making a new project so that you have less typing and repeated inputs.
The solution you linked is mainly useful for already existing ones.

So I kindly ask you to reopen this issue for the above stated reason of the main feature request not being implemented (Yet).

**Comment by Prospector** [comment]
yeah this is partially implemented at best

**Comment by Minenash** [comment]
On the idea of templates, one possible solution would be to have templates be a project status-type. This would also the template to have all the settings a normal project would, would have the project-type specific stuff available, and since templates are for private use, it doesn't need review. And since it's just a project, all the backend would need to do to use a template is clone the project

Ofc it should be in a different section in the dashboard
